src/layouts/about.jsx

```diff
@@ -2,7 +2,186 @@ import React from 'react';
+        always emerging, SignsFive will be a place where d/Deaf/HoH,
+        interpreters and ASL students (and hearing users of ASL) can
+        find signs for STE(A)M terms and contribute to the growing collection.
+        </p>
+      </section>
+      <div className="row py-5">
+        {/* <!--about-our-story--> */}
+        <div className="col-md-6 px-5">
+          <h3 className="tracked pb-2">Our Story</h3>
+          <p>
+            <strong>In early 2017, a group of deaf/hard-of-hearing/deafblind and hearing professionals began meeting weekly at <a href="https://chihacknight.org/">ChiHackNight</a> to discuss what issues that haven’t been addressed well.</strong>
+          </p>
+
+          <p>
+            As a developer/engineer, Jason Salerno shared that he had been frustrated that
+            the recurring problems for existing online/web sign language dictionaries for STE(A)M:
+          </p>
+
+          <ol>
+            <li>
+              Not active anymore because there are no more grants to
+                  support or admins have moved on
+            </li>
+            <li>Not user-friendly nor mobile-friendly
+
+            </li>
+            <li>Not fast enough to adds signs, especially in STE(A)M field where
+                it’s ever changing
+            </li>
+          </ol>
+          <p>
+            Initially called OpenSigns, the group hatched SignsFive, a community-sourced
+            open to anyone in the sign language community to contribute, share and discuss
+            STE(A)M signs.
+          </p>
+
+        </div>
+
+
+        {/* <!--about-our-name--> */}
+        <div className="col-md-6">
+          <h3 className="tracked pb-2">Our Name</h3>
+          <p>
+            <strong>
+              You might be wondering, {'"'}What’s the story behind the name?{'"'}
+            </strong>
+          </p>
+
+          {/* <!-- TODO: Make icons accessible to screenreaders --> */}
+          <p>Short answer: <em>OpenSigns</em> has been taken by a sign shop. 👎</p>
+
+          <p>
+            But after weeks of exploring and bantering, we realized that the number <em>five</em>
+            had been a repetitive theme, referring both to
+            <a href="{http://www.lifeprint.com/asl101/pages-layout/parameters.htm}">{'"'}Five Parameters of ASL{'"'}</a>
+            and the five letters in STE(A)M. Most importantly, the sign for
+            SignsFive was too cool
+            to pass. For the win!
+          </p>
+
+          <div className="pt-1">
+            <section className="bg-dark text-white text-center align-middle py-5">GIF of SignsFive sign</section>
+          </div>
+        </div>
+      </div>
+
+      {/* team */}
+      <header className="container">
+        <div className="py-5 px-5">
+          <h3 className="tracked text-center pb-2">Meet the Team</h3>
+        </div>
+      </header>
+
+      <div className="container">
```

Comment:
This should be its own JSX and also it can be one components and we can use `.map` with it and are it reusable.

Reply:
Separated out. Team is pulled from data.js file imported into a comp.

Reply:
main.css renamed